common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -206,9 +222,24 @@ protected List<IIpcStrategy> getIpcStrategies(final Context applicationContext,
     public @NonNull
     String hello(final @NonNull IIpcStrategy strategy,
                  final @Nullable String minRequestedVersion) throws BaseException {
+        return hello(strategy, minRequestedVersion, CLIENT_MAX_PROTOCOL_VERSION);
+    }
+
+    /**
+     * MSAL-Broker handshake operation.
+     *
+     * @param strategy            an {@link IIpcStrategy}
+     * @param minRequestedVersion the minimum allowed broker protocol version, may be null.
```

Comment:
>aram minRequestedVersion the minimum allowed broker protocol version, may be null [](http://example.com/codeflow?start=9&length=81)

3rd param details

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2008/commits/f91c570c2e8ff147b8ca5d31f480e83c7a44c279